OlaRabbitMqAdminToolkitBundle.php

```diff
@@ -6,4 +6,5 @@
 
 class OlaRabbitMqAdminToolkitBundle extends Bundle
 {
+  public plooup = "
```

Comment:
:thinking:

Reply:
Damn, I'm so stupid, I try to fail the build by introducing a crap in a non tested class ! :D

Reply:
:D